### PR TITLE
Updating repository in prep for 10.4 release.

### PR DESCRIPTION
**Pull request by lbrendanl**
Hey Dave, here are the small doc updates for the upcoming spatial feature.  You are welcome make changes directly to this dev branch or just let me know if you want me to make some updates.  

We won't want to merge this until 10.4 is released.

### CODE REVIEW

File: _posts/2017-08-21-spatial-data-10-4.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: "WDC Spatial Data in Tableau 10.2"
```
Comment (d45) [inline _posts/2017-08-21-spatial-data-10-4.md]: Shouldn't this be Tableau 10.4 ?

File: _posts/2017-08-21-spatial-data-10-4.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+Learn more about how to upgrade to Tableau 10.4 and about the new features available here: [Get Tableau 10.4](https://www.tableau.com/new-features/10.4).
+
+Learn more about GeoJSON here: [GeoJSON documentation](http://geojson.org/).
+
+You can use the following hosted API scripts to take advantage of this new feature: 
+
+```
+https://connectors.tableau.com/libs/tableauwdc-2.3.latest.js
+https://connectors.tableau.com/libs/tableauwdc-2.3.latest.min.js
+```
```
Comment (d45) [inline _posts/2017-08-21-spatial-data-10-4.md]: We need to update some of the GitHub pages  to list the 2.3 version https://github.com/tableau/webdataconnector/blob/master/docs/wdc_library_versions.md

In the compatibility table we need to add WDC 2.3.  I assume that 2.3 is for 10.4 (and later)

File: Examples/js/earthquakeUSGS.js [hunk 3]
```diff
@@ -16,13 +16,8 @@
...
             dataType: tableau.dataTypeEnum.string
         }, {
-            id: "lat",
-            alias: "latitude",
-            dataType: tableau.dataTypeEnum.float
-        }, {
-            id: "lon",
-            alias: "longitude",
-            dataType: tableau.dataTypeEnum.float
+            id: "location",
```
Comment (d45) [inline Examples/js/earthquakeUSGS.js]: OK. I guess the tutorial needs to be updated. The schema example, still shows the longitude and latitude.  
I can dig into this and make the changes.  I"m assuming that I can just clone the dev branch, make and test changes locally, and then make a separate PR to dev. Not sure if those changes can be added to this PR to master, or if that will need to be a separate one.

Comment (lbrendanl) [inline Examples/js/earthquakeUSGS.js]: Ah yes, good catch, I forgot that was used in the tutorial.  

Thank you! That would be awesome.  Yes, actually if you just clone dev, you can commit directly to it and this PR will be updated automatically.

File: _posts/2017-08-21-spatial-data-10-4.md (hunk 2 above)
Comment (lbrendanl) [inline _posts/2017-08-21-spatial-data-10-4.md]: Exactly right. 

And for the tutorial, there are already hosted versions at connectors.tableau.com for 2.3 if you need them.

File: _posts/2017-08-21-spatial-data-10-4.md (hunk 1 above)
Comment (lbrendanl) [inline _posts/2017-08-21-spatial-data-10-4.md]: Ahh yes, copy paste bug.  Thanks for catching!  If you wouldn't mind updating that in your pass through I would appreciate it.

Comment (d45) [inline _posts/2017-08-21-spatial-data-10-4.md]: That's what I though. Yes. I'll update that.

File: Examples/js/earthquakeUSGS.js (hunk 3 above)
Comment (d45) [inline Examples/js/earthquakeUSGS.js]: Will do. Thanks. Just testing things right now to verify the doc changes. I'll make the doc changes next.

File: _posts/2017-08-21-spatial-data-10-4.md (hunk 2 above)
Comment (d45) [inline _posts/2017-08-21-spatial-data-10-4.md]: Cool. Thanks!  Nice of you to put them up there.

**Pull request merged**